packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -324,8 +324,8 @@ class RenderEditable extends RenderBox
     bool paintCursorAboveText = false,
     Offset cursorOffset = Offset.zero,
     double devicePixelRatio = 1.0,
-    ui.BoxHeightStyle selectionHeightStyle = ui.BoxHeightStyle.tight,
-    ui.BoxWidthStyle selectionWidthStyle = ui.BoxWidthStyle.tight,
+    ui.BoxHeightStyle selectionHeightStyle = ui.BoxHeightStyle.max,
```

Comment:
I set this to a reasonable default that wouldn't cause a line with runs of different heights to create "ledges".

Reply:
I'm on board with that. I've seen the "ledges" problem come up before and wondered why we didn't do this in the first place. We can always undo it if it breaks something we're not thinking of.